src/Files.App/ViewModels/ShellViewModel.cs

```diff
@@ -228,8 +227,6 @@ public async Task SetWorkingDirectoryAsync(string? value)
 
 			if (value == "Home" || value == "ReleaseNotes" || value == "Settings")
 				currentStorageFolder = null;
-			else
-				_ = Task.Run(() => jumpListService.AddFolderAsync(value));
```

Comment:
Does this need a replaement from our end?

Reply:
I think that if we don't manually add folders when opening them in Files, they won't get added to the frequent access list.

Reply:
That case, to keep the synchronization, we should update Explorer's jump list, then the watcher (I will create in the next PR) would notice it and update Files' one too.
What you think?